index.js

```diff
@@ -107,6 +107,10 @@ function devMode(enabled) {
   atlas.setDevMode(enabled);
 }
 
+function analyzeId(name, tags) {
```

Comment:
nit: is `analyzeId` a correct name if we pass a name and an object representing tags and not an actual ID (which IIUC is a unique string composed from a name and a set of tags)?

Wouldn't e.g. `validateNameAndTags` be a better name?

Reply:
Sounds good to me. I'll rename it.

Reply:
Thanks!